src/Composer/Installers/EzPlatformInstaller.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace Composer\Installers;
+
+class EzPlatformInstaller extends BaseInstaller
+{
+    protected $locations = array(
+        'meta-assets' => 'web/assets/ezplatform/',
+        'assets' => 'web/assets/ezplatform/{$name}/',
```

Comment:
Given this comes from composer packages, name could also be `'web/assets/package`

But I'm on the fence what is better here, @bdunogier?